src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -235,15 +242,13 @@ def get_sdfg_args(sdfg: dace.SDFG, *args, check_args: bool = False, **kwargs) ->
     dace_shapes = get_shape_args(sdfg.arrays, dace_field_args)
     dace_conn_shapes = get_shape_args(sdfg.arrays, dace_conn_args)
     dace_strides = get_stride_args(sdfg.arrays, dace_field_args)
-    dace_conn_strides = get_stride_args(sdfg.arrays, dace_conn_args)
     dace_offsets = get_offset_args(sdfg, args)
     all_args = {
         **dace_args,
         **dace_conn_args,
         **dace_shapes,
         **dace_conn_shapes,
         **dace_strides,
-        **dace_conn_strides,
```

Comment:
While the removal of the offsets of the connectivities sounds reasonable, I do not get why you also remove the strides here.
I mean bellow you make the change in `add_storage()` to no longer generate the offsets, but I do not see a similar change for the strides (which I thing we should keep).

Reply:
Right, it was half left in my head. Ok, let's keep the strides for the connectivity arrays.

Reply:
What I find a bit strange is, how the hell did it run successfully?

Reply:
Probably because we hard-code stride0 to the local size `max_neighbors`, and we do not use stride1 for indexing because the connectivity table is always 2D.